.travis.yml

```diff
@@ -31,12 +31,16 @@ before_install:
 
 install:
   - |
-    if [[ -v PRODUCT_TESTS || -v HIVE_TESTS ]]; then
-      ./mvnw install $MAVEN_FAST_INSTALL -pl '!presto-docs,!presto-server-rpm'
+    if [[ -v TEST_SPECIFIC_MODULES ]]; then
+      ./mvnw install $MAVEN_FAST_INSTALL -pl $TEST_SPECIFIC_MODULES -am
     fi
   - |
-    if [[ -v TEST_MODULES ]]; then
-      ./mvnw install $MAVEN_FAST_INSTALL -pl $TEST_MODULES -am
+    if [[ -v TEST_OTHER_MODULES || -v HIVE_TESTS ]]; then
```

Comment:
`HIVE_TESTS` only needs `presto-hive-hadoop2` and dependencies, so it can do this:
```shell
-pl presto-hive-hadoop2 -am